data/glossaries/external.yml

```diff
@@ -395,10 +395,10 @@
     Markdown and uses a particular form of Markdown called GitHub Flavored Markdown. See [GitHub Flavored Markdown Spec](https://github.github.com/gfm/) or [Getting started with writing and formatting on GitHub](/articles/getting-started-with-writing-and-formatting-on-github).
 - term: Markup
   description: A system for annotating and formatting a document.
-- term: master
+- term: main
   description: >-
     The default development branch. Whenever you create a Git repository, a
-    branch named "master" is created, and becomes the active branch. In most
+    branch named "main" is created, and becomes the active branch. In most
```

Comment:
> Whenever you create a Git repository, a branch named "master" is created

Unfortunately this isn't true. If you create a directory, go into it, and do `git init` to create a new repository, in most cases you'll find yourself in a branch called `master` - unless you have configured a git template that changes this to something else, like `main`.

I'd suggest we simplify the definition of `main` here to just:

> Typically, the default branch for repositories on GitHub is called `main`.

Reply:
So shall I make a commit with proposal to add separate definition of `main` in the glossary as suggested by you, keeping the definition of `master` intact since it's the current standard in Git?

Reply:
Good point. I think you're right: we shouldn't do away with `master` in the glossary all together. Could you commit another change that alters the definition for `main` and adds an entry for `master`. I think we should simplify the definition of `master` and indicate that it's not a term used at GitHub. How about:

> The default branch in many Git repositories. By default, when you create a new Git repository on the command line a branch called `master` is created. Many tools now use an alternative name for the default branch. For example, when you create a new repository on GitHub the default branch is called `main`.

Reply:
done